language/test/analyze.test.js

```diff
@@ -1,368 +1,29 @@
-
-      sample.program.analyzeSyntaxFromFile(bucketName, fileName, {
-        type: type,
-        language: language
-      }, callback);
-
-      assert.equal(callback.calledOnce, true);
-      assert.deepEqual(callback.firstCall.args, [error]);
-    });
-  });
-
-  describe('main', function () {
-    var options = { type: 'text', language: undefined };
-
-    it('should call analyzeSentimentFromString', function () {
-      var program = getSample().program;
-
-      sinon.stub(program, 'analyzeSentimentFromString');
-      program.main(['sentimentFromString', text]);
-      assert.equal(program.analyzeSentimentFromString.calledOnce, true);
-      assert.deepEqual(program.analyzeSentimentFromString.firstCall.args.slice(0, -1), [text, options]);
-    });
-
-    it('should call analyzeSentimentFromFile', function () {
-      var program = getSample().program;
-
-      sinon.stub(program, 'analyzeSentimentFromFile');
-      program.main(['sentimentFromFile', bucketName, fileName]);
-      assert.equal(program.analyzeSentimentFromFile.calledOnce, true);
-      assert.deepEqual(program.analyzeSentimentFromFile.firstCall.args.slice(0, -1), [bucketName, fileName, options]);
-    });
-
-    it('should call analyzeEntitiesFromString', function () {
-      var program = getSample().program;
-
-      sinon.stub(program, 'analyzeEntitiesFromString');
-      program.main(['entitiesFromString', text]);
-      assert.equal(program.analyzeEntitiesFromString.calledOnce, true);
-      assert.deepEqual(program.analyzeEntitiesFromString.firstCall.args.slice(0, -1), [text, options]);
-    });
-
-    it('should call analyzeEntitiesFromFile', function () {
-      var program = getSample().program;
-
-      sinon.stub(program, 'analyzeEntitiesFromFile');
-      program.main(['entitiesFromFile', bucketName, fileName]);
-      assert.equal(program.analyzeEntitiesFromFile.calledOnce, true);
-      assert.deepEqual(program.analyzeEntitiesFromFile.firstCall.args.slice(0, -1), [bucketName, fileName, options]);
-    });
-
-    it('should call analyzeSyntaxFromString', function () {
-      var program = getSample().program;
-
-      sinon.stub(program, 'analyzeSyntaxFromString');
-      program.main(['syntaxFromString', text]);
-      assert.equal(program.analyzeSyntaxFromString.calledOnce, true);
-      assert.deepEqual(program.analyzeSyntaxFromString.firstCall.args.slice(0, -1), [text, options]);
-    });
-
-    it('should call analyzeSyntaxFromFile', function () {
-      var program = getSample().program;
-
-      sinon.stub(program, 'analyzeSyntaxFromFile');
-      program.main(['syntaxFromFile', bucketName, fileName]);
-      assert.equal(program.analyzeSyntaxFromFile.calledOnce, true);
-      assert.deepEqual(program.analyzeSyntaxFromFile.firstCall.args.slice(0, -1), [bucketName, fileName, options]);
-    });
+describe(`language:analyze`, () => {
+  it(`should have expected exports`, () => {
+    assert.equal(typeof program.analyzeSentimentOfText, `function`);
```

Comment:
Is this necessary? (Won't it "halt and catch fire" if this is missing, or is it better to explicitly check this to save debugging time?)